docs/usage.rst

```diff
@@ -6,15 +6,95 @@ Usage
 Example
 -------
 
-* Run the helloRadiomics example, using sample data provided in ``pyradiomics/data``:
+* PyRadiomics example code and data is available in the `Github repository <https://github.com/Radiomics/pyradiomics>`_
+
+* The sample sample data is provided in ``pyradiomics/data``.
+
+* Use jupyter to run the helloRadiomics example, located in ``pyradiomics/bin/Notebooks``.
+
+* If jupyter is not installed, run the python script alternative (``pyradiomics/bin/helloRadiomics.py``):
+
+  * ``python helloRadiomics.py``
+
+----------------
+Command Line Use
+----------------
+
+* PyRadiomics has 2 commandline scripts, ``pyradiomics`` is for single image feature extraction and ``pyradiomicsbatch``
+  is for feature extraction from a batch of images and segmentations.
+
+* Both scripts can be run directly from a command line window, anywhere in your system.
+
+* To extract features from a single image and segmentation run::
+
+    pyradiomics <path/to/image> <path/to/segmentation>
+
+* To extract features from a batch run::
+
+    pyradiomicsbatch <path/to/input> <path/to/output>
```

Comment:
Trick is how it's defined in the setupscript. There you link your CLI command (e.g. pyradiomics) to the function it should run (in this case, `radiomics.scripts.commandline.main()`), similar for pyradiomics batch. You could also just specify the scripts as such, however, this doesn't allow you to name them differently and it requires the user to add .py in the command window. I thought it more intuitive to hava a command like `pyradiomics`